### PR TITLE
project product card

### PR DESCRIPTION
**Pull request by ZoyaCanepa**
- [DEMO LINK](https://baranovskazoya.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://baranovskazoya.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,64 @@
...
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body class="body">
+    <div
+      class="card"
+      data-qa="card"
+    >
```
Comment (Anna-Borodaienko) [inline src/index.html:17]:
```suggestion
    <div class="card" data-qa="card">
```

You need to use different lines if you have 3 or more properties

File: src/index.html [hunk 2]
```diff
@@ -1,13 +1,64 @@
...
+        src="images/imac.jpeg"
+        alt="imac"
+      >
+      <h2 class="card__name">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+
+      <p class="card__code">
+        Product code: 195434
+      </p>
```
Comment (Anna-Borodaienko) [inline src/index.html:29]:
```suggestion
      <p class="card__code">Product code: 195434</p>
```

File: src/index.html [hunk 3]
```diff
@@ -1,13 +1,64 @@
...
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <p class="card__rating-reviews">
+          Reviews: 5
+        </p>
```
Comment (Anna-Borodaienko) [inline src/index.html:42]:
```suggestion
        <p class="card__rating-reviews">Reviews: 5</p>
```

File: src/index.html [hunk 4]
```diff
@@ -1,13 +1,64 @@
...
+      </div>
+
+      <p class="card__price">
+        <span class="card__price-name">
+          Price:
+        </span>
+        <span class="card__price-value">
+        $2,199
+        </span>
```
Comment (Anna-Borodaienko) [inline src/index.html:51]:
```suggestion
        <span class="card__price-name">Price:</span>
        <span class="card__price-value">$2,199</span>
```

File: src/index.html [hunk 5]
```diff
@@ -1,13 +1,64 @@
...
+      </p>
+
+
+      <a
+      class="card__link"
+      href="#"
+      data-qa="hover"
+      >
+         Buy
+      </a>
```
Comment (Anna-Borodaienko) [inline src/index.html:61]:
```suggestion
      data-qa="hover"
      >
        Buy
      </a>
```

File: src/reset.css [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+
+* {
+  box-sizing: border-box;
+}
+
+body,
+h2,
+p {
+  margin: 0;
+  padding: 0;
```
Comment (Anna-Borodaienko) [inline src/reset.css:15]: These tags haven't default paddings, you don't need to reset them

File: src/style.css [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+
+.card__price-value {
+  color: var(--color-accent);
+  font-weight: 700;
+  font-size: 16px;
+}
+
+.card__link {
+
+  padding: 10px 70px;
```
Comment (Anna-Borodaienko) [inline src/style.css:68]:
```suggestion
.card__link {
  padding: 10px 70px;
```

**Review by MaksimusCRS — APPROVED** [review]
GJ

**Pull request closed without merging**